webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/reports/ExecutionBatchTest.scala

```diff
@@ -1095,11 +1095,11 @@ class ExecutionBatchTest extends Specification {
     }
     "when allow duplicated, duplicate messages is ignored" in {
       val res = ExecutionBatch.checkExpectedComponentWithReports(expectedComponent, duplicated, ReportType.Missing, PolicyMode.Enforce, UnexpectedReportInterpretation(Set(UnexpectedReportBehavior.AllowsDuplicate)))
-      res.compliance === ComplianceLevel(success = 2)
+      res.compliance === ComplianceLevel(success = 1, unexpected = 2)
```

Comment:
You need to delete that test, and the whole concept of `UnexpectedReportInterpretation(Set(UnexpectedReportBehavior.AllowsDuplicate))` I think, since it doesn't make sense without syslog (or is it also used to control the property for different unexpected, like the one produced by loops ? If so, only remove `UnexpectedReportBehavior.AllowsDuplicate` case)

Reply:
Ah yes, it is, with `UnboundVarValues`, so keep `UnexpectedReportBehavior` but remove `AllowsDuplicate`